### PR TITLE
[REACTOS] Fix some 'Ukranian' typos

### PR DESCRIPTION
**Pull request by SergeGautherie**

### CODE REVIEW

File: sdk/include/psdk/twain.h [hunk 1]
```diff
@@ -1268,7 +1268,7 @@ typedef struct {
 #define TWLG_SWEDISH              TWLG_SWE
 #define TWLG_THAI                 84
 #define TWLG_TURKISH              85
-#define TWLG_UKRANIAN             86
+#define TWLG_UKRAINIAN            86
```
Comment (HBelusca) [inline sdk/include/psdk/twain.h:1271]: Nope! You are breaking compatibility with official headers
https://github.com/twain/twain-specification/blob/master/versions/2.1/twain.h#L1427

File: sdk/include/ddk/usbprotocoldefs.h [hunk 2]
```diff
@@ -541,7 +541,7 @@ typedef struct _USBHUBPORTDATA {
 #define USB_LANG_TURKISH                                    0x1f
 #define USB_LANG_URDU                                       0x20
 #define USB_LANG_INDONESIAN                                 0x21
-#define USB_LANG_UKRANIAN                                   0x22
+#define USB_LANG_UKRAINIAN                                  0x22
```
Comment (HBelusca) [inline sdk/include/ddk/usbprotocoldefs.h:544]: No! You are breaking compatibility with official headers:
https://github.com/Aekras1a/Labs/blob/master/Labs/WinDDK/7600.16385.1/inc/ddk/USBProtocolDefs.h#L742

Comment (Saibamen) [reply sdk/include/ddk/usbprotocoldefs.h:544]:
```suggestion
#define USB_LANG_UKRANIAN                                   0x22
```

File: sdk/include/psdk/twain.h (hunk 1 above)
Comment (Saibamen) [reply sdk/include/psdk/twain.h:1271]:
```suggestion
#define TWLG_UKRANIAN             86
```

**Comment by ThFabba** [comment]
Changes requested more than two weeks ago. Please reopen if/when comments are addressed.

**Pull request closed without merging**